page-components/tables.md

```diff
@@ -4,6 +4,1217 @@ title:  Tables
+        <tr>
+            <th class="u-w20pct">
+                Type
+            </th>
+            <th class="u-w55pct">
+                Title
+            </th>
+            <th class="u-w25pct">
+                Date
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td data-label="Type">
+                <span class="cf-icon cf-icon-speech-bubble"></span> Blog
+            </td>
+            <td data-label="Title">
+                Three things to do before closing
+            </td>
+            <td data-label="Date">
+                AUG 5, 2015
+            </td>
+        </tr>
+        <tr>
+            <td data-label="Type">
+                <span class="cf-icon cf-icon-bullhorn"></span> News
+            </td>
+            <td data-label="Title">
+                Electronic Mortgage Closings Can Benefit Consumers
+            </td>
+            <td data-label="Date">
+                AUG 5, 2015
+            </td>
+        </tr>
+        <tr>
+            <td data-label="Type">
+                <span class="cf-icon cf-icon-speech-bubble"></span> Blog
+            </td>
+            <td data-label="Title">
+                National Day of Civic Hacking 2015
+            </td>
+            <td data-label="Date">
+                JUL 29, 2015
+            </td>
+        </tr>
+    </tbody>
+</table>
+{:/nomarkdown}
+</div>
+
+<div class="content-50 content-first">
+### Pagination
+
+Tables with over 20 rows can be paired with pagination.
+</div>
+
+<div class="content-50 content-last">
+{::nomarkdown}
+<table class="table__stack-on-small" style="width: 100%;">
+    <thead>
+        <tr>
+            <th class="u-w20pct">
+                Type
+            </th>
+            <th class="u-w55pct">
+                Title
+            </th>
+            <th class="u-w25pct">
+                Date
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td data-label="Type">
+                <span class="cf-icon cf-icon-speech-bubble"></span> Blog
+            </td>
+            <td data-label="Title">
+                Three things to do before closing
+            </td>
+            <td data-label="Date">
+                AUG 5, 2015
+            </td>
+        </tr>
+        <tr>
+            <td colspan="3" style="text-align: center; font-weight: bold;">
+                (18 more rows)
+            </td>
+        </tr>
+        <tr>
+            <td data-label="Type">
+                <span class="cf-icon cf-icon-speech-bubble"></span> Blog
+            </td>
+            <td data-label="Title">
+                National Day of Civic Hacking 2015
+            </td>
+            <td data-label="Date">
+                JUL 29, 2015
+            </td>
+        </tr>
+    </tbody>
+</table>
+<nav class="post-pagination pagination" role="navigation" aria-label="Pagination" data-ajax-action="posts-paginated.html">
+    <a class="btn btn__super btn__disabled pagination_prev">
```

Comment:
Looks like `.btn__super` is in the pagination example at https://cfpb.github.io/capital-framework/components/cf-pagination/, too.

I think the issue might be squeezing the pagination element into a narrow column. Not sure there's a good solution for that.